config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -177,8 +177,8 @@ periodics:
       - --deployment=node
       - --gcp-project-type=node-e2e-project
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml
-      - --node-test-args= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/perf-cos-81-12871-103-0-config.yaml
```

Comment:
sorry, I'll be more explicit, I think the filename should be `perf-image-config.yaml`

Reply:
haha :sweat_smile: fixed it!